### DISCUSSION TITLE
Manutenzione ambiente di produzione ANSC

### DISCUSSION THREAD

**Discussion by rvinzani** [italia/ansc#985] [Supporto servizi cooperativi]
Si comunica che oggi 6 agosto, dalle ore 17 alle ore 19:30 potrebbero verificarsi dei brevi disservizi nell'ambiente di produzione di ANSC per operazione di manutenzione del sistema

**Comment by mttfranci** [comment]
L'aggiornamento dell'ambiente di produzione è terminato.

Saluti

Assistenza ANSC
